observability/src/test/java/io/grpc/observability/interceptors/LogHelperTest.java

```diff
@@ -595,17 +753,24 @@ public InputStream stream(byte[] value) {
 
   // Copied from internal
   static final class IoUtils {
-    /** maximum buffer to be read is 16 KB. */
+
+    /**
+     * maximum buffer to be read is 16 KB.
+     */
```

Comment:
It will be good to avoid unnecessary changes like this. If this was done by the IDE consider setting the correct options on the IDE

Reply:
Done.